pkgdb2/api/admin.py

```diff
@@ -151,7 +151,7 @@ def api_admin_actions():
             status=status,
             count=True,
         )
-    except pkgdblib.PkgdbException, err:  # pragma: no cover
+    except pkgdblib.PkgdbException as err:  # pragma: no cover
```

Comment:
> So, imho, either we are consistent and fix it everywhere or figure out more precisely why the tests were failing
￼There are two different exceptions, one with Bugzilla it is name. This one caused the test failure. The changes to the exception withouth Bugzilla in its name was there to make it more consistent.